docs/06-releasing.md

```diff
@@ -0,0 +1,21 @@
+# Releasing
+
+*This is a WIP document*
+
+The following steps are needed to release the app:
+
+* Tag latest git commit on master
+* Create a release in Sentry and [upload source maps](05-error-reporting.md).
+* Create a release in Jira so we can reference to this later.
+* Create a entry in [CHANGELOG.md](../CHANGELOG.md)
+* Decide whether we want to send the app to testing prior to releasing
```

Comment:
`testing` means beta-channel? Or what does it mean? 🤔

Reply:
Not further specified as of now. Testing internally, commercial testing etc.